ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -141,23 +141,10 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
                             const syncResourceKey = new URLSearchParams(this.props.history.location.search).get('deploy');
                             const tab = new URLSearchParams(this.props.history.location.search).get('tab');
 
```

Comment:
There's a filter to show orphaned resources. Does this mean they will not be accounted for?

Reply:
Thanks for catching this Keith, that part of code got lost in resolving merge conflicts. will add it back